### ISSUE TITLE
503 server error

### ISSUE THREAD

**Issue by dev-lusaja** [camptocamp/docker-consul-demo#3]
Hello,
I set the same example in a virtual server everything seems okay, I manage to display the dashboard consul and proxy, but when up to app test hello-world at times I responds with a 503 error which does not happen when I answered the example in a local environment.

**Comment by jacohend** [comment]
Yeah, this example seems to intermittently produce 503 errors for no reason with a very simple 8080 service.

**Comment by jacohend** [comment]
@raphink Any ideas?

**Comment by raphink** [comment]
I haven't really used this poc in months. We're now using rancher, which provides both service discovery and load balancing services.

**Comment by seyfer** [comment]
But how to fix 503 error in this stack ?

**Comment by dev-lusaja** [comment]
Any idea ??
